Bottom: c5ceb76922688ccc69387b32240d1c7baf973fa8
Top:    b2ef334fb25d57042ba7b6ec17c13281f6fe4e28
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-03 12:55:58 +0000

lesstif: Silence two set but unused variable warnings


---

diff --git a/src/hid/lesstif/netlist.c b/src/hid/lesstif/netlist.c
index 598d106..79ee647 100644
--- a/src/hid/lesstif/netlist.c
+++ b/src/hid/lesstif/netlist.c
@@ -298,7 +298,7 @@ netlist_button (Widget parent, char *name, char *string,
 static int
 build_netlist_dialog ()
 {
-  Widget b_sel, b_unsel, b_find, b_ripup, b_rat_on, b_rat_off, l_ops;
+  Widget b_sel, b_unsel, b_find, b_rat_on, l_ops;
   XmString ops_str;
 
   if (!mainwind)
@@ -318,9 +318,9 @@ build_netlist_dialog ()
 			     (XtCallbackProc)nbcb_std_callback, (void *)nbcb_rat_on);
 
   n = 0;
-  b_rat_off = netlist_button (netlist_dialog, "rat_off", "Disable for rats",
-			      0, NLB_FORM, b_rat_on, 0,
-			      (XtCallbackProc)nbcb_std_callback, (void *)nbcb_rat_off);
+  netlist_button (netlist_dialog, "rat_off", "Disable for rats",
+                  0, NLB_FORM, b_rat_on, 0,
+                  (XtCallbackProc)nbcb_std_callback, (void *)nbcb_rat_off);
 
   n = 0;
   b_sel = netlist_button (netlist_dialog, "select", "Select",
@@ -339,9 +339,9 @@ build_netlist_dialog ()
 
 
   n = 0;
-  b_ripup = netlist_button (netlist_dialog, "ripup", "Rip Up",
-			    0, b_rat_on, b_find, 0,
-			    (XtCallbackProc)nbcb_ripup, 0);
+  netlist_button (netlist_dialog, "ripup", "Rip Up",
+                  0, b_rat_on, b_find, 0,
+                  (XtCallbackProc)nbcb_ripup, 0);
 
   n = 0;
   stdarg (XmNbottomAttachment, XmATTACH_WIDGET);
